### DISCUSSION TITLE
bump for list email plugin

### DISCUSSION THREAD

**Discussion by thommangooo** [trobfab/fabrik#512] [General]
I note that the list email plug-in hasn't made its way into the first beta drop and just wanna say that I use it extensively and look forward to its return.  
Cheers and thanks for the great work!

**Comment by trob** [comment]
List email plugin is pretty complex.

**Reply by thommangooo** [reply]
My users find it extremely valuable.  Lots of time saved.  It's a real customer delighter!

**Comment by thommangooo** [comment]
I've pretty much got everything working in my site, but it turns out I use this plug-in quite a bit.  Without it, I'm back to square 1.3ish.  This plugin has been a constant customer delighter as it turns tedious tasks into very simple actions.  I've got to unwind a lot of customer value if it's not going to make the cut.  Fingers crossed.
